app/internal/page/donor/remove_attorney.go

```diff
@@ -44,6 +44,10 @@ func RemoveAttorney(logger Logger, tmpl template.Template, lpaStore LpaStore) pa
 
 			if data.Form.RemoveAttorney == "yes" && data.Errors.None() {
 				lpa.Attorneys.Delete(attorney)
+				if len(lpa.Attorneys) == 1 {
```

Comment:
Doing this means I don't have to check that there are multiple attorneys in other places. But I think I'm going to play around with refactoring `Attorneys` to include the decisions so all the logic will be in one place, but in a different PR.